packages/tools/devtools/devtools-view/src/components/data-visualization/TreeHeader.tsx

```diff
@@ -22,20 +22,30 @@ export interface TreeHeaderProps extends HasLabel {
 	inlineValue?: React.ReactElement | string;
 
 	// TODO: metadata
```

Comment:
Let's get rid of this comment, since we're addressing it. Happy to have this done!